trin-core/src/portalnet/overlay_service.rs

```diff
@@ -232,7 +233,7 @@ pub struct OverlayService<TContentKey> {
     /// The protocol identifier.
     protocol: ProtocolId,
     /// A queue of peers that require regular ping to check connectivity.
```

Comment:
Would you mind adding a one-liner description about the `HashSetDelay` datatype? how it works/why we're using it